client/ayon_houdini/api/lib.py

```diff
@@ -896,40 +896,51 @@ def get_current_context_template_data_with_entity_attrs():
     return template_data
 
 
-def set_review_color_space(opengl_node, review_color_space="", log=None):
+def set_review_color_space(node, review_color_space="", log=None):
     """Set ociocolorspace parameter for the given OpenGL node.
 
-    Set `ociocolorspace` parameter of the given OpenGl node
+    Set `ociocolorspace` parameter of the given node
     to to the given review_color_space value.
     If review_color_space is empty, a default colorspace corresponding to
     the display & view of the current Houdini session will be used.
 
+    Note: 
+        This function expects nodes of type `opengl` or `flipbook`.
+
     Args:
-        opengl_node (hou.Node): ROP node to set its ociocolorspace parm.
+        node (hou.Node): ROP node to set its ociocolorspace parm.
         review_color_space (str): Colorspace value for ociocolorspace parm.
         log (logging.Logger): Logger to log to.
     """
 
     if log is None:
         log = self.log
 
+    if node.type().name() not in {"opengl", "flipbook"}:
+        log.warning(
+            "Type of given node {} not allowed."
+            " only types `opengl` and `flipbook` are allowed."
+            .format(node.type().name())
+        )
+
     # Set Color Correction parameter to OpenColorIO
-    colorcorrect_parm = opengl_node.parm("colorcorrect")
-    if colorcorrect_parm.eval() != 2:
-        colorcorrect_parm.set(2)
+    colorcorrect_parm = node.parm("colorcorrect")
+    if colorcorrect_parm.evalAsString() != "ocio":
+        idx = colorcorrect_parm.menuItems().index("ocio")
+        colorcorrect_parm.set(idx)
         log.debug(
-            "'Color Correction' parm on '{}' has been set to"
-            " 'OpenColorIO'".format(opengl_node.path())
+            "'Color Correction' parm on '{}' has been set to '{}'"
+            .format(node.path(), colorcorrect_parm.menuLabels()[idx])
         )
 
-    opengl_node.setParms(
+    node.setParms(
         {"ociocolorspace": review_color_space}
     )
 
     log.debug(
         "'OCIO Colorspace' parm on '{}' has been set to "
         "the view color space '{}'"
-        .format(opengl_node, review_color_space)
+        .format(node.path(), review_color_space)
```

Comment:
just wondering why you don't use `node` (hou.Node) here as the previous code used `opengl_node` which happened to be hou.Node.

Reply:
I thought it was already solved.

Reply:
for consistency, I turned this one into `node.path()` instead of just `node`
other log messages are using `node.path()`